shared/js/ui/models/site.es6.js

```diff
@@ -1,6 +1,5 @@
 const Parent = window.DDG.base.Model
 const constants = require('../../../data/constants')
-const trackerPrevalence = require('../../../data/tracker_lists/prevalence')
```

Comment:
This is in the tds data now

Reply:
Interesting that it was included w/o being used 🤷‍♂